frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -403,3 +414,26 @@ export const getSecretAnnotations = (annotation: KeyValuePair) => {
   }
   return annotations;
 };
+
+export const pipelinesTab = (kindObj: K8sKind) => {
+  switch (kindObj.kind) {
```

Comment:
I think for this to be an accurate usage, we need to cover the situation where they pass valid models that don't have unique values (the "root" pages).

```
    case PipelineModel.kind:
    case TaskModel.kind:
    case EventListenerModel.kind:
      return '';
```

That way there is a distinction between `null` (not a pipeline tab) and `''` a "root" pipeline tab. This would be my expectation when using a function called `pipelinesTab` to get a tab extension for the pipelines pages.